packages/EasyCore/src/Bridge/Symfony/ApiPlatform/Command/ApiResourceAndSimpleDataPersisterMaker.php

```diff
@@ -26,7 +31,7 @@ public static function getCommandName(): string
     public function configureCommand(Command $command, InputConfiguration $inputConfig): void
     {
         $command
-            ->setDescription('Create ApiResource + its SimpleDataPersister')
+            ->setDescription(self::getCommandDescription())
```

Comment:
@itorgov I think we can completely remove this call now that we have the `getCommandDescription()` method.

Reply:
Done.